hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/AssignmentManager.java

```diff
@@ -2028,4 +2036,41 @@ private void addToPendingAssignment(final HashMap<RegionInfo, RegionStateNode> r
   MasterServices getMaster() {
     return master;
   }
+
+  public Map<byte[], Pair<ServerName, Set<ServerName>>> getProblematicRegions() {
```

Comment:
how do we get from this method to the master UI?

we need a way to get the same information via a cli, either hbck1 or shell.

Reply:
Added the UI in new patch.

Reply:
I thought hbck1 already have the ability to find these problematic regions?